core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -39,7 +40,17 @@ object BrokerApiVersionsCommand {
   def execute(args: Array[String], out: PrintStream): Unit = {
     val opts = new BrokerVersionCommandOptions(args)
     val adminClient = createAdminClient(opts)
-    val brokerMap = adminClient.listAllBrokerVersionInfo()
+
+    // Wait until AdminClient#findAllBrokers returns a non-empty list of brokers
```

Comment:
@lindong28 : Good idea.  I think using a do...while loop is better.

@hachikuji: Yeah, let's have a method in AdminClient to do this... it seems simple enough.